mypy/test/testextensions.py

```diff
@@ -0,0 +1,103 @@
+import sys
+import pickle
+from unittest import TestCase, main, skipUnless, SkipTest
+from extensions.mypy_extensions import TypedDict
+
+
+class BaseTestCase(TestCase):
+
+    def assertIsSubclass(self, cls, class_or_tuple, msg=None):
+        if not issubclass(cls, class_or_tuple):
+            message = '%r is not a subclass of %r' % (cls, class_or_tuple)
+            if msg is not None:
+                message += ' : %s' % msg
+            raise self.failureException(message)
+
+    def assertNotIsSubclass(self, cls, class_or_tuple, msg=None):
+        if issubclass(cls, class_or_tuple):
+            message = '%r is a subclass of %r' % (cls, class_or_tuple)
+            if msg is not None:
+                message += ' : %s' % msg
+            raise self.failureException(message)
+
+
+PY36 = sys.version_info[:2] >= (3, 6)
+
+PY36_TESTS = """
+Label = TypedDict('Label', [('label', str)])
+
+class Point2D(TypedDict):
+    x: int
+    y: int
+
+class LabelPoint2D(Point2D, Label): ...
+"""
+
+if PY36:
+    exec(PY36_TESTS)
+
+
+class TypedDictTests(BaseTestCase):
+
+    def test_basics_iterable_syntax(self):
+        # Check that two iterables allowed
+        Emp = TypedDict('Emp', [('name', str), ('id', int)])
```

Comment:
This is rather by accident. The implementation just takes an arbitrary iterable as an argument, e.g., a dictionary ``TypedDict('Employee', {'name': str, 'id': int})``. List of tuples etc. also work. Basically everything that could be given to a normal ``dict`` works. Of course we could prohibit everything apart from a real ``dict``, but I think that would be redundant. Anyway, I just removed this test, since I agree that it is unnecessary.